src/RtpFecQueue.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "Video.h"
+
+typedef struct _RTPFEC_QUEUE_ENTRY {
+    PRTP_PACKET packet;
+
+    uint64_t queueTimeMs;
```

Comment:
I don't really think it makes much sense to track this in the FEC queue. We'll automatically move on to the next frame when it arrives.